staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go

```diff
@@ -436,14 +471,121 @@ func TestProxyUpgrade(t *testing.T) {
 			ExpectError:  true,
 			ExpectCalled: false,
 		},
+		"valid hostname + CABundle + egress selector": {
```

Comment:
I think we only need the "valid hostname + CABundle + egress selector" and the "valid hostname + CABundle + egress selector non working" cases. The cases with invalid hostname or invalid CA do not tell us if the egress selector code works as expected.

Reply:
done